models/project/project.go

```diff
@@ -217,7 +217,7 @@ func (u *ProjectStruct) Lists() []*list.ListStruct {
 	return slice
 }
 
-func (u *ProjectStruct) NoneList() *list.ListStruct {
+func (u *ProjectStruct) NoneList() (*list.ListStruct, error) {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.NoneList should have comment or be unexported